Source/ConversationV1/Models/OutputData.swift

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright IBM Corporation 2017
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import Foundation
+
+/** An output object that includes the response to the user, the nodes that were hit, and messages from the log. */
+public struct OutputData {
+
+    /// Up to 50 messages logged with the request.
+    public var logMessages: [LogMessage]
+
+    /// An array of responses to the user.
+    public var text: [String]
+
+    /// An array of the nodes that were triggered to create the response.
+    public var nodesVisited: [String]?
+
+    /// Additional properties associated with this model.
+    public var additionalProperties: [String: JSON]
+
+    /**
+     Initialize a `OutputData` with member variables.
+
+     - parameter logMessages: Up to 50 messages logged with the request.
+     - parameter text: An array of responses to the user.
+     - parameter nodesVisited: An array of the nodes that were triggered to create the response.
+
+     - returns: An initialized `OutputData`.
+    */
+    public init(logMessages: [LogMessage], text: [String], nodesVisited: [String]? = nil, additionalProperties: [String: JSON] = [:]) {
+        self.logMessages = logMessages
+        self.text = text
+        self.nodesVisited = nodesVisited
+        self.additionalProperties = additionalProperties
+    }
+}
+
+extension OutputData: Codable {
+
+    private enum CodingKeys: String, CodingKey {
+        case logMessages = "log_messages"
+        case text = "text"
+        case nodesVisited = "nodes_visited"
+        static let allValues = [logMessages, text, nodesVisited]
+    }
+
+    public init(from decoder: Decoder) throws {
+        let container = try decoder.container(keyedBy: CodingKeys.self)
+        let dynamic = try decoder.container(keyedBy: DynamicKeys.self)
```

Comment:
The text highlighting here seems to imply that `dynamic` is a Swift keyword.  Not sure we need to do anything about this, but I thought I should mention it.

Reply:
Yup! It is a Swift keyword, although it's only ever used as a modifier in front of a declaration (e.g. `dynamic var` or `dynamic function`). Xcode is smart enough not to highlight this instance.

It's easy enough to change, though. I updated the template, regenerated the models, and pushed a commit.